CHANGELOG.md

```diff
@@ -1,5 +1,128 @@
+- <!-- 26 --> [Accordion] Migrate AccordionActions to emotion (#24538) @vinyldarkscratch
+- <!-- 27 --> [Accordion] Migrate AccordionDetails to emotion (#24539) @vinyldarkscratch
+- <!-- 30 --> [Link] Migrate to emotion (#24529) @praveenkumar-kalidass
+- <!-- 31 --> [Accordion] Migrate to emotion (#24518) @vinyldarkscratch
+- <!-- 34 --> [Backdrop] Migrate to emotion (#24523) @vinyldarkscratch
+- <!-- 41 --> [Grid] Add system props (#24499) @mnajdova
+- <!-- 42 --> [Icon] Migrate to emotion (#24516) @vinyldarkscratch
+- <!-- 44 --> [Collapse] Migrate to emotion (#24501) @vinyldarkscratch
+- <!-- 47 --> [SvgIcon] Migrate to emotion (#24506) @oliviertassinari
+- <!-- 48 --> [Avatar] Migrate AvatarGroup to emotion (#24452) @praveenkumar-kalidass
+- <!-- 49 --> [Box] Add back system props (#24485) @mnajdova
+- <!-- 50 --> [Alert] Migrate AlertTitle to emotion (#24448) @povilass
+- <!-- 54 --> [Alert] Migrate to emotion (#24442) @kutnickclose
+- <!-- 59 --> [l10n] Improve Hebrew translation (#24449) @eladmotola
+- <!-- 61 --> [Checkbox][Switch] Document defaultChecked (#24446) @praveenkumar-kalidass
+- <!-- 62 --> [AppBar] Migrate to emotion (#24439) @povilass
+- <!-- 64 --> [l10n] Improve German translation (#24436) @lukaselmer
+- <!-- 65 --> [Button][Badge] Support custom colors and sizes (#24408) @mnajdova
+- <!-- 70 --> [Grid] Fix horizontal scrollbar and nested dimensions (#24332) @greguintow
+- <!-- 73 --> [Grid] Migrate to emotion (#24395) @mnajdova
+- <!-- 74 --> [Badge] Fix TS custom variants (#24407) @mnajdova
+
+### `@material-ui/lab@`
+
+- <!-- 32 --> [DatePicker] Remove unnecessary wrapping dom node (#24533) @mxsph
+- <!-- 68 --> [DateRangePicker] Remove variant prop override for Textfield (#24433) @praveenkumar-kalidass
+- <!-- 77 --> [lab] Reflect draft pattern of picker value in implementation (#24367) @eps1lon
+
+### `@material-ui/styled-engine@`
+
+- <!-- 67 --> [styled-engine] Rename StylesProvider to StyledEngineProvider (#24429) @mnajdova
+
+### `@material-ui/system@`
+
+- <!-- 36 --> [system] Fix handling of null-ish values (#24530) @oliviertassinari
+
+### `@material-ui/unstyled@`
+
+- <!-- 72 --> [unstyled] Convert composeClasses to TypeScript (#24396) @eps1lon
+
+### `@material-ui/utils@`
+
+- <!-- 20 --> [utils] `useEventCallback` `args` defaults to `unknown[]` (#24564) @eps1lon
+- <!-- 69 --> [utils] Fix requirePropFactory to merge validators (#24423) @mnajdova
+
+### core
+
+- <!-- 29 --> [core] Improve pseudo classes overrides error (#24535) @mnajdova
+- <!-- 45 --> [core] Fix styleProps to always contain all props (#24505) @mnajdova
+- <!-- 56 --> [core] Do not forward classes prop by default in experimentalStyled (#24451) @mnajdova
+- <!-- 57 --> [core] Pass styleProps on all slots in the styled() components (#24454) @mnajdova
+- <!-- 58 --> [core] Batch small changes (#24445) @oliviertassinari
+- <!-- 79 --> [core] Normalize generating declaration files (#24411) @eps1lon
+
+### docs
```

Comment:
I think that docs should come before core, core is mainly for "hard-core" developers that want to know about the infrastructure work going on.

Reply:
done